clientSide/src/components/NavBar/Navbar.jsx

```diff
@@ -0,0 +1,32 @@
+import {Navbar, Nav, Container} from 'react-bootstrap'
+import "./NavBar.css"
+import '@fortawesome/fontawesome-free/css/all.min.css';
+export const NavBar = () =>{
+    return (
+
+
+<Navbar className="nav-bar" expand="lg" sticky="top">
+  <Container>
+    <Navbar.Brand href="#home">Scancar</Navbar.Brand>
+    <Navbar.Toggle aria-controls="basic-navbar-nav" />
+     <Navbar.Collapse id="basic-navbar-nav"> 
+      <Nav className="ml-auto">    
+          
+            <Nav.Link href="find">Find booking</Nav.Link>
+            <Nav.Link href="book">Book</Nav.Link>
+            <Nav.Link href="pricing">Overview</Nav.Link>
+            <Nav.Link href="pricing">Release</Nav.Link>
+            <Nav.Link href="pricing">Request</Nav.Link>    
+            <Nav.Link href="logOut"> <i className="fas fa-sign-out-alt"></i> </Nav.Link>     
```

Comment:
It is not the final version but since the PR is mandatory we just pushed a dummy version :)

Reply:
I know. I realize it may not be useful, but I figure I'd might as well comment. Use it or ignore it.

You may already have this intention, but paths should be the same as the menu name. Paths are visible to users remember 🙂

Reply:
Perfect, thanks 🤸🏼‍♂️, a group member is working on the React-Bootstrap documentation rn! Tak for feedback